templates/docs/examples/patterns/navigation/_script-sliding.js

```diff
@@ -0,0 +1,85 @@
+const initNavigationSliding = () => {
+  document.querySelector('.js-menu-button').addEventListener('click', function (e) {
+    if (window.location.hash === '#navigation') {
```

Comment:
Have a look at nav example with search: https://vanilla-framework-4767.demos.haus/docs/examples/patterns/navigation/search-dark

Try to base this JS on that example.